tracing-log/CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+# 0.2.0 (October 23rd, 2023)
+
+This release contains two breaking changes: the removal of the `env_logger`
+and `trace_logger` features. Below are the suggested migration paths:
+
+- `env_logger`: users should use [`tracing_subscriber::fmt::Subscriber`]
+or [`tracing_subscriber::fmt::Layer`] with the [`Targets`] or [`EnvFilter`]
+filters instead.
+- `trace_logger`: users should use the `tracing` crate's "log" feature flag instead.
+
+### Breaking Changes
+
+- Remove `env_logger` feature ([#2771])
+- Remove `trace_logger` feature ([#2771])
```

Comment:
can we maybe mention somewhere in here that `env_logger` was removed as part of mitigation for a RustSec advisory?